pkg/cli/admin/ocpcertificates/certregen/command.go

```diff
@@ -25,11 +25,8 @@ var (
 	`)
 
 	regenerateSignersExample = templates.Examples(`
-		# Regenerate all known signing certificates on the cluster.
-		oc adm certificates regenerate-signers -A --all
-
 		# Regenerate the signing certificate contained in a particular secret.
-		oc adm certificates regenerate-signers -n openshift-kube-apiserver-operator loadbalancer-serving-signer
+		oc adm ocp-certificates regenerate-top-level -n openshift-kube-apiserver-operator loadbalancer-serving-signer
```

Comment:
here should be `oc adm ocp-certificates regenerate-top-level -n openshift-kube-apiserver-operator secret/loadbalancer-serving-signer`

Reply:
Updated

Reply:
@ardaguclu   please remove the `line breaks` in the example.

Reply:
@zhouying7780 thanks. I think, no need to run all the CI jobs just for this newline. Because also other examples have it as well.